components/Grid.js

```diff
@@ -26,9 +26,10 @@ Box.propTypes = {
 };
 
 export const Flex = styled(Box)(
-  {
+  props => ({
     display: 'flex',
-  },
+    gap: props.gap,
```

Comment:
I don't get why we need this line, aren't all props passed down automatically?

Reply:
It should, as `gap` is a valid flex-box property that is not being captured by the composable prop functions.